android/src/main/java/com/revenuecat/purchases/hybridcommon/mappers/LogHandlerWithMapping.kt

```diff
@@ -0,0 +1,30 @@
+package com.revenuecat.purchases.hybridcommon.mappers
+
+class LogHandlerWithMapping(
+    private val callback: (logData: Map<String, String>) -> Unit
+) : com.revenuecat.purchases.LogHandler {
```

Comment:
Could we add an import instead of using the full name?

Reply:
Oh yes, I had named this class LogHandler and this is a leftover